client/src/components/App.jsx

```diff
@@ -1,32 +1,31 @@
 /* eslint-disable react/destructuring-assignment */
 import React from 'react';
-// import axios from 'axios';
+import axios from 'axios';
 import PhotoGallery from './PhotoGallery.jsx';
-import Save from './Save.jsx';
-import house from '../sample_data.js';
 import Modal from './Modal.jsx';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      pictures: house.pictures,
+      pictures: [],
       showModal: false,
       clickedPicture: null,
     };
     this.clickPicture = this.clickPicture.bind(this);
     this.exitModal = this.exitModal.bind(this);
   }
 
-  // componentDidMount() {
-  //   axios({
-  //     method: 'get',
-  //     url: '/house/110',
-  //   })
-  //     .then((house) => {
-  //       this.setState({pictures: house.data.pictures}, () => console.log('set state', house.data.pictures));
-  //     });
-  // }
+  componentDidMount() {
+    axios({
+      method: 'get',
+      url: '/house/115',
```

Comment:
should you make this more general now? i assume you don't want to hardcode this any more?